src/pages/AddItem.js

```diff
@@ -31,57 +43,81 @@ const AddItem = () => {
 
   return (
     <>
-      <AddItemHeader />
-      <form onSubmit={handleSubmit}>
-        <label>
-          Add Item:
-          <input
-            value={name}
-            placeholder="apples"
-            type="text"
-            onChange={handleChange}
-          />
-        </label>
-        <input type="submit" value="Add Item" />
-        <div>
-          <label>
-            <input
-              type="radio"
-              id={expectedPurchase.soon}
-              value={expectedPurchase.soon}
-              checked={expectedPurchase.soon === nextExpectedPurchase}
-              onChange={handleSelect}
-            />
-            Soon
-          </label>
-        </div>
-        <div>
-          <label>
-            <input
-              type="radio"
-              id={expectedPurchase.verySoon}
-              value={expectedPurchase.kindOfSoon}
-              checked={expectedPurchase.kindOfSoon === nextExpectedPurchase}
-              onChange={handleSelect}
-            />
-            Kind of Soon
-          </label>
-        </div>
-        <div>
-          <label>
-            <input
-              type="radio"
-              id={expectedPurchase.notSoon}
-              value={expectedPurchase.notSoon}
-              checked={expectedPurchase.notSoon === nextExpectedPurchase}
-              onChange={handleSelect}
-            />
-            Not Soon
-          </label>
-        </div>
-      </form>
+      <AppHeader showBackArrow={true}>Add Item to List</AppHeader>
+
+      <Container textAlign="center" style={{ paddingBottom: 50 }}>
+        <Segment>
+          <div>
+            <form onSubmit={handleSubmit}>
+              <Input
+                disabled={!list ? 'disabled' : null}
+                value={name}
+                size="large"
+                placeholder="Name of Item"
+                type="text"
+                onChange={handleChange}
+              />
+              {error && name && <ItemError name={name} />}
+              <p>How soon do you expect to buy this item again?</p>
+              <List>
+                <List.Item>
+                  <List.Content>
+                    <label>
```

Comment:
Any reason to not use 'Label` from semantic ui?

Reply:
The semantic label is under the Form component and I didn't think to convert the forms to `Form`.